test/function/samples/nested-plugin/_config.js

```diff
@@ -0,0 +1,21 @@
+const plugin = [
+	{
+		name: 'nested-plugin-1',
+		transform(code) {
+			return code.replace('foo = 1', 'foo = 2');
+		}
+	},
+	{
+		name: 'nested-plugin-2',
+		transform(code) {
+			return code.replace('answer = 41', 'answer = 42');
+		}
+	}
+];
```

Comment:
Maybe add a `false` or `undefined` to the array to test the nested filtering. The idea being that plugins can use simple switches to deactivate nested plugins. Rollup should just ignore this.